### PR TITLE
Edit index and functionalites file

### PR DESCRIPTION
**Pull request by basirking**
**Here we:**
**-Use webpack to bundle JavaScript.**
**- Use proper ES6 syntax.**
**-Use ES6 modules to write modular JavaScript.**
**-Add add and remove functionality.**
**-Add edit function to be able to edit.**

### CODE REVIEW

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @basirking, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [ ] I cannot leave comments there so I am leaving them here.  Kindly implement event listener on the checkbox (change), you are using `click`.
- [ ] Your local storage is not working properly. When I add data it adds properly and when I remove it by using `clear all completed` buttons and again add some data the index value is not working as it is repeating. So kindly fix it.
### Error
your

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ const ClearAllBtn = () => {
   });
   let count = 0;
   const data = Array.from(localData).filter(i => i.completed === false);// eslint-disable-line
-  data.map(i => i.index = count += 1); // eslint-disable-line
+  data.map(i.index = count += 1); // eslint-disable-line
```
Comment (descholar-ceo) [inline src/index.js:86]: :tada:

**Review by descholar-ceo — APPROVED** [review]
Hi @basirking , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:+1: Fixed the requested changes :heavy_check_mark: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (basirking) [reply src/index.js:86]: if I don't disable the Eslint in this line my linters check will not pass.

Comment (basirking) [reply src/index.js:86]: @descholar-ceo thanks for your excellent feedback I really appreciate it.
I wish you all the best! 💯

**Pull request merged**